ui/pages/confirmations/components/confirm/info/shared/transaction-details/transaction-details.tsx

```diff
@@ -63,7 +63,7 @@ const RecipientRow = () => {
   );
 };
 
-const MethodDataRow = () => {
+export const MethodDataRow = () => {
```

Comment:
These wouldn't have to be exported and other components coupled to them, if we had a pluggable component such as `BaseTransactionDetails` that accepted `children`?